docker-compose-openmrs.yml

```diff
@@ -0,0 +1,108 @@
+services:
+
+  # OpenMRS 3 Backend
+  openmrs:
+    depends_on:
+      mysql:
+        condition: service_started
+      env-substitution:
+        condition: service_completed_successfully
+    environment:
+      OMRS_CONFIG_MODULE_WEB_ADMIN: "true"
+      OMRS_CONFIG_AUTO_UPDATE_DATABASE: "true"
+      OMRS_CONFIG_CREATE_TABLES: "true"
+      OMRS_CONFIG_CONNECTION_SERVER: mysql
+      OMRS_CONFIG_CONNECTION_DATABASE: openmrs
+      OMRS_CONFIG_CONNECTION_USERNAME: ${OPENMRS_DB_USER:-openmrs}
+      OMRS_CONFIG_CONNECTION_PASSWORD: ${OPENMRS_DB_PASSWORD:-openmrs}
+    healthcheck:
+      test:
+        - CMD
+        - curl
+        - "-f"
+        - "http://localhost:8080/openmrs/"
+      timeout: 5s
+    image: openmrs/openmrs-reference-application-3-backend:${O3_BACKEND_TAG:-nightly}
+    labels:
+      traefik.enable: "true"
+      traefik.http.routers.openmrs.rule: "Host(`${O3_HOSTNAME}`) && PathPrefix(`/openmrs`)"
+      traefik.http.routers.openmrs.entrypoints: "websecure"
+      traefik.http.services.openmrs.loadbalancer.server.port: 8080
+      traefik.http.routers.openmrs.middlewares: openmrs-spa-redirectregex
+      traefik.http.middlewares.openmrs-spa-redirectregex.redirectregex.regex: https://${O3_HOSTNAME}/openmrs/spa
+      traefik.http.middlewares.openmrs-spa-redirectregex.redirectregex.replacement: https://${O3_HOSTNAME}/openmrs/spa/home
+    networks:
+      - ozone
+      - web
+    ports:
+      - "8080:8080"
+    restart: unless-stopped
+    volumes:
+      - "openmrs-data:/openmrs/data"
+      - "${OPENMRS_OWAS_PATH:-openmrs-owas}:/openmrs/distribution/openmrs_owas/"
+      - "${OPENMRS_CORE_PATH:-openmrs-core}:/openmrs/distribution/openmrs_core/"
+      - "${OPENMRS_MODULES_PATH:-openmrs-modules}:/openmrs/distribution/openmrs_modules/"
+      - "${OPENMRS_CONFIG_PATH:-openmrs-config}:/openmrs/distribution/openmrs_config/"
+      - "${OPENMRS_CONFIG_CHECKSUMS_PATH:-openmrs-config-checksums}:/openmrs/data/configuration_checksums"
+      - "${OPENMRS_PROPERTIES_PATH}:/etc/properties/"
```

Comment:
We need to add https://github.com/ozone-his/ozonepro-docker/blob/b31a8ad5856ec31d021b33e6fc0789e616b4e4e6/docker-compose.yml#L84